pdc/apps/compose/tests.py

```diff
@@ -839,6 +848,29 @@ def test_clone_overridesrpm_with_error_keys(self):
                                     format='json')
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
+    def test_clone_and_verify_release_rpm_mapping(self):
+        # Source release can pass with release_mapping method work
+        response1 = self.client.get(reverse('releaserpmmapping-detail',
+                                            args=['release-1.0', 'bash']))
+        self.assertEqual(response1.status_code, status.HTTP_200_OK)
+        # Before clone, target release can't work with release_mapping method
+        target_response = self.client.get(reverse('release-list'), release_id='release-3.0')
+        self.assertEqual(target_response.status_code, status.HTTP_200_OK)
+        target_release_id = target_response.data['results'][1]['release_id']
+
+        response2 = self.client.get(reverse('releaserpmmapping-detail',
+                                            args=[target_release_id, 'bash']))
+        self.assertEqual(response2.status_code, status.HTTP_404_NOT_FOUND)
+        response3 = self.client.post(reverse('overridesrpmclone-list'),
+                                     {'source_release_id': 'release-1.0',
+                                     'target_release_id': target_release_id},
+                                     format='json')
+        self.assertEqual(response3.status_code, status.HTTP_201_CREATED)
```

Comment:
Maybe it's not very necessary use response1, response2, response3, looks like just response is fine. It seems all response will be used only once.